config/config/wc-config.yaml

```
@@ -252,3 +261,17 @@ networkPolicies:
 
   alertmanager:
     enabled: true
+
+## Open Policy Agent Gatekeeper configuration
+gatekeeper:
+  allowUserCRDs:
+    enabled: false
+    enforcement: deny
+    extraCRDs: []
```

Maybe add a comment mentioning that seledsecrets/mongodb are "automatically" added via the `user.monogodb/sealed` settings
```suggestion
    extraCRDs: []
```